fiatconnect-api.md

```diff
@@ -527,6 +639,8 @@ used for the transfer.
   - An ISO 3166-1 country code representing the country where the quote should be requested for.
 * `region`: {`string`}
   - An optional ISO 3166-2 subdivision code representing a region within the provided country.
+* `address`: {`string`} [REQUIRED]
+  - An [EIP-55](https://eips.ethereum.org/EIPS/eip-55) formatted address, representing the Celo address of the user to get the quote for.
```

Comment:
for contract-owned accounts should this be the EOA or their contract address? the contract, right?

Reply:
Yep, clarified!